app/models/ability.rb

```diff
@@ -34,6 +36,11 @@ def checkin_checkout_user_permissions(_user)
     can %i[create update read], CheckpointUnit
   end
 
+  def checkin_checkout_user_manager_permissions(_user)
+    can %i[manage], Checkpoint
```

Comment:
Do they need to be able to change all checkpoints? There is also a export ability
```suggestion
    can %i[export], CheckpointUnit
```